src/view/components/decisionScopes.jsx

```diff
@@ -152,12 +69,114 @@ export const getSettings = ({ values }) => {
   return undefined;
 };
 
-export const validationSchema = {
+const validationSchema = object().shape({
   decisionScopesDataElement: string().when("decisionsInputMethod", {
     is: DATA_ELEMENT,
     then: string().matches(
       singleDataElementRegex,
       "Please specify a data element"
     )
   })
+});
+
+const DecisionScopes = () => {
+  const { initInfo } = useContext(ExtensionViewContext);
+  const initialValues = getInitialValues({ initInfo });
+
+  return (
+    <ExtensionViewForm
+      initialValues={initialValues}
+      getSettings={getSettings}
+      validationSchema={validationSchema}
+      render={({ formikProps }) => {
+        const {
+          decisionsInputMethod,
+          decisionScopesArray
+        } = formikProps.values;
+
+        return (
+          <Fragment>
```

Comment:
You may know this already, but I think you can just do `<>` instead of importing `Fragment`. Either one works for me though.